IRPQ.Rmd

```diff
@@ -0,0 +1,1152 @@
+</br>
+
+
+
+###  Assigned PQ
+
+Once the PQ has been assigned by Stats Gov, it is then sent out to the relevant teams, relating to the topic areas. Each topic area will generate one email containing all the PQs for that allocation session. 
+
+The email contains information on how to proceed, the SG contact email, as well as the deadline dates. 
+
+**The 'Answer expected' date is for when the question will need to be sent to the relevant MSP. It is not the deadline for PHS to respond to the SG.**
+
+
+#### Example
+
+```{r echo=FALSE, fig.align='center', out.width="100%"}
+knitr::include_graphics("images/irpq-allocatedTeamEmail.png")
+```
+
+
+
+### Knowledge Check 
+
+```{r PQ-AllocationsQuiz}
+quiz(
+  question("You have received a PQ which is not related to your service area, and has only been sent to your team. What should you do? (select ALL that apply)",
+           answer("Contact Stats Gov and inform them of the error.", correct = TRUE),
+           answer("If possible, forward the answer to the appropriate PHS team.", correct = TRUE),
+           answer("Contact Scottish Government, and tell them PHS cannot assist", message="You should not contact Scottish Government, as this PQ has been incorrectly assigned to your team."),
+           answer("Do nothing and wait for Scottish Government to ask for assistance", message="You should make every effort to ensure the PQ is received by the correct team, and that Stats Gov are informed of this error."),
+           allow_retry = TRUE,
+           random_answer_order = TRUE
+           ),
+  question("If you receive a PQ with Action 'For Information', you can ignore it as you will not need to provide a response.",
+           answer("TRUE", message = "Teams should still read the PQs and take further action if necessary."),
+           answer("FALSE", correct = TRUE),
+           allow_retry = TRUE,
+           random_answer_order = TRUE
+           )
+)
+```
+In the next section, we will cover what you should consider when you receive a PQ.
+
+
+## PQ - Types & Considerations
+
+The template used to respond to a PQ can be found in Appendix 3 of the <a href="https://spark.publichealthscotland.org/corporate-guidance/statistical-governance/parliamentary-questions/" target="_blank">Parliamentary Questions Protocol Guide.</a>
+
+Within this template, you can see two key sections:
+
+* **Draft Answer** - this is where you will supply your answer to the PQ.
+
+* **Background Note** - this is additional/background information which could provide greater understanding or references.
+
+Before you start to respond to the PQ, there are a number of considerations to bear in mind.
+
+
+### Considerations 
+
+In the event you receive a PQ, you should carefully consider whether or not you are able to provide a response, what it may involve, and what you may need to discuss with your, or another, team.
+
+The following are a list of considerations which you should discuss with your team:
+
+### <blank>
```

Comment:
In order to allow for users to click through to each section rather than have all the #### sections shown.

If this is not allowed, I could change the #### sections to ### sections.